selfdrive/car/toyota/carstate.py

```diff
@@ -142,7 +145,12 @@ def update(self, cp):
     self.a_ego = float(v_ego_x[1])
     self.standstill = not v_wheel > 0.001
 
-    if self.CP.carFingerprint in TSS2_CAR:
+    if self.CP.carFingerprint in NO_DSU_CAR and self.CP.carFingerprint not in TSS2_CAR: # Vehicle that needs offsetting
+      self.angle_steers = cp.vl["STEER_TORQUE_SENSOR"]['STEER_ANGLE'] - self.offset
+      if not self.isoffset:
```

Comment:
not initialized

Reply:
I removed the bool declaration in int due to:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/7257172/62092416-136d6480-b243-11e9-8871-6de488f16f12.png">

Anyway, pushed.